protocol/app/process/transactions.go

```diff
@@ -34,6 +34,24 @@ func init() {
 	if slices.Min[[]int](txIndicesAndOffsets) != lastOtherTxLenOffset {
 		panic("lastTxLenOffset is not the lowest offset")
 	}
+	if slices.Max[[]int](txIndicesAndOffsets)+1 != firstOtherTxOffset {
+		panic("firstOtherTxOffset is <= the maximum offset")
+	}
+	txIndicesForMinTxsCount := []int{
+		proposedOperationsTxIndex,
+		updateMarketPricesTxLenOffset + minTxsCount,
+		addPremiumVotesTxLenOffset + minTxsCount,
+		acknowledgeBridgesTxLenOffset + minTxsCount,
+	}
+	if minTxsCount != len(txIndicesForMinTxsCount) {
+		panic("minTxsCount does not match expected count of Txs.")
+	}
+	if lib.ContainsDuplicates(txIndicesForMinTxsCount) {
+		panic("Overlapping indices and offsets defined for Txs.")
+	}
+	if minTxsCount != firstOtherTxOffset-lastOtherTxLenOffset {
+		panic("Unexpected gap between firstOtherTxOffset and lastOtherTxLenOffset which is greater than minTxsCount")
+	}
```

Comment:
Wait won't this panic with our current values?
Also I think use comparison rather than equality. That is:
```
if minTxsCount < firstOtherTxOffset-lastOtherTxLenOffset {
```